### DISCUSSION TITLE
Module "vue-advance-chat" has no exported member 'register'

### DISCUSSION THREAD

**Discussion by greyson-uberlegen** [advanced-chat/advanced-chat-components#358] [Q&A] [answered]
Hi, i'm new to this library.
Currently i'm facing this issue, mind to give some idea on this?

**Comment by antoine92190** [comment, marked as answer]
Which framework are you using? Which version of the framework? Which library version are you using?
If you don't give any information, I cannot help you.

**Reply by greyson-uberlegen** [reply]
Hi Sorry i miss out the important information.
I'm using vue 2 and "vue-advanced-chat": "^2.0.3",

**Reply by antoine92190** [reply]
Did you add `isCustomElement: tagName => tagName === 'vue-advanced-chat'` in you `vue.config.js` file?
If it's still not working, can you share you repo so I can have a look?

**Reply by greyson-uberlegen** [reply]
Hi, I'm sorry for the late response.
Currently i'm using the ABP template, in my vue.config.js it using configureWebpack.
May i know how to configure the <sub>isCustomElement: tagName => tagName === 'vue-advanced-chat' </sub> in vue.config.js?

**Reply by antoine92190** [reply]
Here is the official guide: https://vuejs.org/guide/extras/web-components.html#example-vue-cli-config
